views/stats.ejs

```diff
@@ -693,6 +693,28 @@ const seaCreatures = [
     }
 ];
 
+const pet_milestones = [
```

Comment:
Could be changed to an object with the ids as object values.

Reply:
What do you mean exactly ?